src/components/progress-report/workflow/progress-report-workflow.repository.ts

```diff
@@ -180,8 +152,8 @@ export class ProgressReportWorkflowRepository extends DtoRepository(
       ])
       .return<{
         id: ID;
-        email: string;
-        roles: Role[];
+        email: string | null;
```

Comment:
Let's revert this type change, and change the optional match on the email address to a required one.
Let the DB do the filtering for us.